### PR TITLE
Fix test with xclim 0.29

### PR DESCRIPTION
**Pull request by huard**

### CODE REVIEW

File: tests/test_bias_correction.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import xarray as xr
+import xclim.sdba
 import xclim.sdba as sdba
```
Comment (Zeitsperre) [inline tests/test_bias_correction.py:3]: ?

**Review by Zeitsperre — APPROVED** [review]

File: tests/test_bias_correction.py (hunk 1 above)
Comment (huard) [reply tests/test_bias_correction.py:3]: TBH, I don't even know why this test is in RavenPy... it's essentially testing xclim, so apart from giving some pointers on how to use sdba, this is not very useful. I suspect this is a building block for future stuff, but let's not spend too much time on this for now.

**Comment by huard** [comment]
I'll wait before merging and make sure Raven is working as expected.

File: tests/test_bias_correction.py (hunk 1 above)
Comment (Zeitsperre) [reply tests/test_bias_correction.py:3]: I imagine we could think of this test as a sanity check for the xclim API in Ravenpy. We do  similar tests in xclim for CLISOPS to ensure that the core functions we rely on are still working as expected.

**Pull request merged**